tools/format.py

```diff
@@ -88,10 +88,18 @@ def register(command):
         default=False,
         help="Skip formatting the cmake",
     )
+    command.add_argument(
+        "-A",
+        "--format-all",
+        dest="format_all",
+        action="store_true",
+        default=False,
+        help="Format all files, including files not changed since master",
```

Comment:
Please specify in the help how it interacts with the `skip` flags. Possibly even change the name from `format_all`/`--format-all` to something which makes more sense when considering those flags.